### PR TITLE
feature ZEN-27190: service definition for dedicated reporting-zopes

### PR DESCRIPTION
**Pull request by corntoole**
- cloned Zope server service for reports
- changed zproxy-nginx.conf to route report request to Zope-reports
- changed Zenoss.resmgr service definitions accordingly

### CODE REVIEW

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
+{
+    "CPUCommitment": 1,
+    "Command": "su - zenoss -c \"/opt/zenoss/bin/runzope\" ",
+    "Commands": {
```
Comment (jplouis) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: Don't need all the commands from the zope definition

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+            "FileName": "/opt/zenoss/etc/zodb_db_main.conf",
+            "Owner": "zenoss:zenoss",
+            "Permissions": "0664"
+        },
+        "/opt/zenoss/etc/zodb_db_session.conf": {
+            "FileName": "/opt/zenoss/etc/zodb_db_session.conf",
+            "Owner": "zenoss:zenoss",
+            "Permissions": "0664"
+        },
+        "/opt/zenoss/etc/zope.conf": {
```
Comment (jplouis) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: Using the same location for zope.conf as the zope service will lead to conflicts when running dev environments in zendv.

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        },
+        {
+            "filters": [
+                "supervisord"
+            ],
+            "path": "/opt/zenoss/log/audit.log",
+            "type": "zenossaudit"
+        }
+    ],
+    "Name": "Zope",
```
Comment (jplouis) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: Should this change to zenreport?

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json (hunk 1 above)
Comment (corntoole) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: I removed them

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json (hunk 2 above)
Comment (corntoole) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: I added a different configuration file and updated the startup command for zenreports.

File: services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json (hunk 3 above)
Comment (corntoole) [inline services/Zenoss.resmgr.lite/Zenoss/User Interface/zenreports/service.json]: I changed it

File: services/Zenoss.resmgr.lite/-CONFIGS-/opt/zenoss/zproxy/conf/zproxy-nginx.conf [hunk 4]
```diff
@@ -107,6 +105,8 @@ http {
             add_header X-XSS-Protection "1; mode=block";
         }
 
+        include zopereports-proxy.conf;
```
Comment (jplouis) [inline services/Zenoss.resmgr.lite/-CONFIGS-/opt/zenoss/zproxy/conf/zproxy-nginx.conf]: zope-static.conf should be added and this removed.

Comment (corntoole) [inline services/Zenoss.resmgr.lite/-CONFIGS-/opt/zenoss/zproxy/conf/zproxy-nginx.conf]: I made that change

**Pull request merged**